app/js/controllers/foyer/ressources/ressources.js

```diff
@@ -83,7 +83,12 @@ angular.module('ddsApp').controller('FoyerRessourcesCtrl', function($scope, $sta
         return result;
     };
 
-    $scope.individusVM = _.map($scope.situation.individus, function(individu) {
+    var demandeur = SituationService.getDemandeur($scope.situation);
+    var conjoint = SituationService.getConjoint($scope.situation);
+    var sortedIndividus = (conjoint ? [demandeur, conjoint] : [demandeur])
+        .concat(SituationService.getEnfants($scope.situation));
```

Comment:
I would have simply put the last parenthesis on the same line as the dot (opening paren followed by newline, I expect a newline before the closing paren).
But this is very good as is  :)